CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+### 2.21.3
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.9](https://fluentbit.io/announcements/v1.8.9/)
+* Amazon CloudWatch Logs for Fluent Bit 1.6.4
+* Amazon Kinesis Streams for Fluent Bit 1.8.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Same as `2.21.2`, this release includes the following fixes for AWS customers that we are working on getting accepted upstream:
+* Bug - Fix return value from `tls_net_read` [fluentbit:4098](https://github.com/fluent/fluent-bit/issues/4098)
+* Bug - Downgrade `mbedtls` to 2.24.0 to fix the performance regression issue in `mbedtls` 2.26.0 [fluentbit:4110](https://github.com/fluent/fluent-bit/issues/4110)
+* Bug - Resolve IMDSv1 fallback error introduced in 2.21.0 [aws-for-fluent-bit:259](https://github.com/aws/aws-for-fluent-bit/issues/259)
+
+Important Note:
+* A security vulnerability was found in [amazonlinux](https://access.redhat.com/security/cve/CVE-2021-43527) which we use as base image to our `aws-for-fluent-bit` image. This new image will be based on an updated version of amazonlinux that resolves this CVE. It is highly recommended to upgrade your existing workload or run new workload with this latest version (2.21.3) of `aws-for-fluent-bit` image.
```

Comment:
I thought we agreed we wouldn't put the "highly recommended" warning? And let customers decide how scared to be.

Reply:
The forced push looks like it didn't make it through.

Reply:
I edited this locally but the forced push looks like it didn't make it through.